test/functional/cmdLineTests/jvmtitests/build.xml

```diff
@@ -75,10 +75,26 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-ex
 						<property name="excludeFile" value="" />
 					</else>
 				</if>
+				<!-- Java21 and up tests-->
+				<if>
+					<not>
+						<matches string="${JDK_VERSION}" pattern="^(8|11|17)$$" />
+					</not>
```

Comment:
This doesn't work for Java 8 because that version is handled above by lines 57-63.

Reply:
Fixed in https://github.com/eclipse-openj9/openj9/pull/19183.